kdenlive/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "2.1"
+
+services:
+  server:
+    image: linuxserver/kdenlive:421.12.3
```

Comment:
Could you add a multi-arch Docker image digest here? You can get this by running: `docker pull linuxserver/kdenlive:421.12.3`

Reply:
done changes for this